src/include/miopen/sqlite_db.hpp

```diff
@@ -27,7 +27,9 @@
 
 #include <miopen/config.h>
 
-#if MIOPEN_ENABLE_SQLITE
+#if !MIOPEN_ENABLE_SQLITE
```

Comment:
I think it is better to produce a CMake error in this case. It would be seen before compilation.

Reply:
The idea is that this header should not be included when MIOPEN_ENABLE_SQLITE=Off. CMake can't detect such cases ;)

Reply:
It would have been much simpler, to simply wrap this header in the `#if / #endif` instead of adding the same  to every site where this header was included.

@atamazov @averinevg